MapboxAndroidDemo/src/main/res/layout/activity_traffic_plugin.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
seperate `<android.support.constraint.ConstraintLayout ` and `xmlns:android="http://schemas.android.com/apk/res/android"` onto different lines so the indentions proper.